apps/meteor/app/livechat/server/lib/Helper.ts

```diff
@@ -201,6 +201,8 @@ export const createLivechatInquiry = async ({
 		visitor: { _id, username, department, status, activity },
 	});
 
+	const contactId = await getContactIdByVisitorId(_id);
```

Comment:
Is this actually needed? I'm not seeing where you're using the contactId from the inquiry.